src/backend/joanie/core/factories.py

```diff
@@ -752,6 +752,19 @@ class Meta:
     student_signed_on = None
     organization_signed_on = None
 
+    @factory.post_generation
+    def signature_backend_reference(self, create, extracted):
+        """
+        define signature_backend_reference as dummy lexpersona backend
+        """
+
+        self.signature_backend_reference = extracted
+        if not extracted and (
+            self.student_signed_on or self.submitted_for_signature_on
+        ):
+            self.signature_backend_reference = f"wfl_fake_dummy_demo_dev_{self.id}"
+        self.save()
```

Comment:
I'll exec the save into the if statement. We do not want to exec this method if we do not change something.